apps/test/unit/lib/ui/PopUpMenuTest.js

```diff
@@ -38,9 +38,7 @@ describe('PopUpMenu', () => {
       </MenuBubble>
     );
     wrapper
-      .find(PopUpMenu.Item)
-      .first()
-      .children()
+      .find('div[onClick]')
       .first()
       .simulate('click');
```

Comment:
Root cause here: [`children()` now has a slightly different meaning.](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#children-now-has-slightly-different-meaning) To fix, I just found a more expressive way to find the element we wanted to click, that happens to be compatible with 2.x and 3.x.